PSP coverage tests fail to build on the MCP750/VxWorks 6.9 target

**Describe the bug**
When building for MCP750 with all unit/coverage tests enabled, an error occurs:

```
In file included from /home/jphickey/cfecfs/github/psp/fsw/mcp750-vxworks/src/cfe_psp_exception.c:46:
/home/jphickey/cfecfs/github/psp/unit-test-coverage/ut-stubs/override_inc/arch/ppc/vxPpcLib.h:32:1: error: "_PPC_FPSCR_UE" redefined
In file included from /opt/WindRiver/vxworks-6.9/target/h/types/vxArch.h:148,
                 from /opt/WindRiver/vxworks-6.9/target/h/types/vxANSI.h:65,
                 from /opt/WindRiver/vxworks-6.9/target/h/limits.h:42,
                 from /opt/WindRiver/gnu/4.3.3-vxworks-6.9/x86-linux2/bin/../../lib/gcc/powerpc-wrs-vxworks/4.3.3/include-fixed/limits.h:122,
                 from /opt/WindRiver/gnu/4.3.3-vxworks-6.9/x86-linux2/bin/../../lib/gcc/powerpc-wrs-vxworks/4.3.3/include-fixed/syslimits.h:7,
                 from /opt/WindRiver/gnu/4.3.3-vxworks-6.9/x86-linux2/bin/../../lib/gcc/powerpc-wrs-vxworks/4.3.3/include-fixed/limits.h:11,
                 from /home/jphickey/cfecfs/github/psp/unit-test-coverage/ut-stubs/inc/PCS_basetypes.h:26,
                 from /home/jphickey/cfecfs/github/psp/unit-test-coverage/ut-stubs/inc/PCS_stdio.h:5,
                 from /home/jphickey/cfecfs/github/psp/unit-test-coverage/ut-stubs/override_inc/stdio.h:15,
                 from /home/jphickey/cfecfs/github/psp/fsw/mcp750-vxworks/src/cfe_psp_exception.c:39:
/opt/WindRiver/vxworks-6.9/target/h/arch/ppc/archPpc.h:187:1: error: this is the location of the previous definition
```

**To Reproduce**
Prepare using `make SIMULATION=ppc-vxworks6.9 ENABLE_UNIT_TESTS=true prep` and sample_defs config
Then run `make`

**Expected behavior**
Build should succeed

**System observed on:**
`gs582w-cfelnx` build machine with MCP750 target

**Reporter Info**
Joseph Hickey, Vantage Systems, Inc.

--- a/unit-test-coverage/ut-stubs/inc/PCS_arch_ppc_vxPpcLib.h
+++ b/unit-test-coverage/ut-stubs/inc/PCS_arch_ppc_vxPpcLib.h
@@ -4,6 +4,29 @@
 
 #include <PCS_basetypes.h>
 
+/*
+ * When building the coverage test on an actual VxWorks target,
+ * these symbols are indirectly provided via the system limits.h file.
+ * This results in a redefinition issue if the custom PCS definitions are used.
+ */
+#if defined(_VXWORKS_OS_) && defined(__PPC__)
+
+#define PCS_PPC_MSR_EE      _PPC_MSR_EE
+#define PCS_PPC_MSR_FP      _PPC_MSR_FP
+#define PCS_PPC_MSR_ME      _PPC_MSR_ME
+#define PCS_PPC_MSR_FE0     _PPC_MSR_FE0
+#define PCS_PPC_MSR_FE1     _PPC_MSR_FE1
+#define PCS_PPC_MSR_DR      _PPC_MSR_DR
+
+#define PCS_PPC_FPSCR_VE    _PPC_FPSCR_VE
+#define PCS_PPC_FPSCR_OE    _PPC_FPSCR_OE
+#define PCS_PPC_FPSCR_NI    _PPC_FPSCR_NI
+#define PCS_PPC_FPSCR_ZE    _PPC_FPSCR_ZE
+#define PCS_PPC_FPSCR_XE    _PPC_FPSCR_XE
+#define PCS_PPC_FPSCR_UE    _PPC_FPSCR_UE
+
+#else
+
 /* ----------------------------------------- */
 /* constants normally defined in vxPpcLib.h */
 /* ----------------------------------------- */
@@ -21,6 +44,8 @@
 #define PCS_PPC_FPSCR_ZE    0x2208
 #define PCS_PPC_FPSCR_XE    0x2210
 #define PCS_PPC_FPSCR_UE    0x2220
+
+#endif
 
 
 /* ----------------------------------------- */

--- a/unit-test-coverage/ut-stubs/override_inc/arch/ppc/vxPpcLib.h
+++ b/unit-test-coverage/ut-stubs/override_inc/arch/ppc/vxPpcLib.h
@@ -14,6 +14,13 @@
 
 #include <PCS_arch_ppc_vxPpcLib.h>
 
+/*
+ * When building the coverage test on an actual VxWorks target,
+ * these symbols are indirectly provided via the system limits.h file.
+ * This results in a redefinition issue if the custom PCS definitions are used.
+ */
+#if !defined(_VXWORKS_OS_) || !defined(__PPC__)
+
 /* ----------------------------------------- */
 /* mappings for declarations in vxPpcLib.h */
 /* ----------------------------------------- */
@@ -31,6 +38,7 @@
 #define _PPC_FPSCR_XE   PCS_PPC_FPSCR_XE
 #define _PPC_FPSCR_UE   PCS_PPC_FPSCR_UE
 
+#endif
 
 #define vxTimeBaseGet   PCS_vxTimeBaseGet
 

